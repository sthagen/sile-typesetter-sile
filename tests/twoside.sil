\begin[papersize=a8,class=book]{document}
\use[module=packages.retrograde,target=v0.15.0]
\neverindent
% Note folio suppression on some pages are part of test
\begin[command=stuff]{define}
% Deliberately push the frame to overflowing but without leaving the frame
% See issue #454 and PR #1547
\eject
\\eject then stuff just past
\skip[height=11bs]
\glue[width=38spc]the gills
\end{define}
Leading content
\open-double-page
\\open-double-page
\eject
\\eject
\open-double-page
\\open-double-page
\stuff
\open-double-page
\\open-double-page
\open-spread
\\open-spread
\stuff
\open-spread[odd=false]
\\open-spread[odd=false]
\open-spread[double=false]
\\open-spread[double=false]
\open-spread
\\open-spread
\end{document}
