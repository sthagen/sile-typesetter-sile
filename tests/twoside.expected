Set paper size 	147.4015764	209.7637818
Begin page
Mx 	12.2343
My 	31.9010
Set font 	Gentium Plus;10;400;;normal;;;LTR;
T	47 72 68 71 76 81 74 w=32.2754	(Leading)
Mx 	47.1575
T	70 82 81 87 72 81 87 w=31.9385	(content)
Mx 	12.2343
My 	43.9010
Mx 	67.1537
My 	187.1052
T	20 w=4.6924	(1)
New page
Mx 	20.6362
My 	24.3326
Mx 	75.5555
My 	187.0564
T	21 w=4.6924	(2)
New page
Mx 	12.2343
My 	32.1451
T	63 w=4.6924	(\)
Mx 	16.9267
T	82 83 72 81 w=20.4639	(open)
Mx 	37.3906
T	16 w=3.3691	(-)
Mx 	40.7597
T	71 82 88 69 79 72 w=27.9346	(double)
Mx 	68.6943
T	16 w=3.3691	(-)
Mx 	72.0634
T	83 68 74 72 w=19.3848	(page)
Mx 	67.1537
My 	187.0564
T	22 w=4.6924	(3)
New page
Mx 	20.6362
My 	32.1451
T	63 w=4.6924	(\)
Mx 	25.3286
T	72 77 72 70 87 w=19.6387	(eject)
Mx 	20.6362
My 	44.1451
My 	56.1451
Mx 	75.5555
My 	187.1052
T	23 w=4.6924	(4)
New page
Mx 	12.2343
My 	32.1451
T	63 w=4.6924	(\)
Mx 	16.9267
T	82 83 72 81 w=20.4639	(open)
Mx 	37.3906
T	16 w=3.3691	(-)
Mx 	40.7597
T	71 82 88 69 79 72 w=27.9346	(double)
Mx 	68.6943
T	16 w=3.3691	(-)
Mx 	72.0634
T	83 68 74 72 w=19.3848	(page)
Mx 	67.1537
My 	186.9978
T	24 w=4.6924	(5)
New page
Mx 	20.6362
My 	32.1451
T	63 w=4.6924	(\)
Mx 	25.3286
T	72 77 72 70 87 w=19.6387	(eject)
Mx 	47.6109
T	87 75 72 81 w=19.0967	(then)
Mx 	69.3512
T	86 87 88 747 w=18.8232	(stuff)
Mx 	90.8180
T	77 88 86 87 w=15.1904	(just)
Mx 	108.6520
T	83 68 86 87 w=17.1924	(past)
Mx 	104.3179
My 	176.1451
T	87 75 72 w=13.5791	(the)
Mx 	119.8055
T	74 76 79 79 86 w=16.8701	(gills)
Mx 	75.5555
My 	187.1248
T	25 w=4.6924	(6)
New page
Mx 	12.2343
My 	32.1451
T	63 w=4.6924	(\)
Mx 	16.9267
T	82 83 72 81 w=20.4639	(open)
Mx 	37.3906
T	16 w=3.3691	(-)
Mx 	40.7597
T	71 82 88 69 79 72 w=27.9346	(double)
Mx 	68.6943
T	16 w=3.3691	(-)
Mx 	72.0634
T	83 68 74 72 w=19.3848	(page)
Mx 	12.2343
My 	44.1451
Mx 	67.1537
My 	186.9099
T	26 w=4.6924	(7)
New page
Mx 	20.6362
My 	24.3326
New page
Mx 	12.2343
My 	32.1451
T	63 w=4.6924	(\)
Mx 	16.9267
T	82 83 72 81 w=20.4639	(open)
Mx 	37.3906
T	16 w=3.3691	(-)
Mx 	40.7597
T	86 83 85 72 68 71 w=27.5293	(spread)
Mx 	67.1537
My 	187.0564
T	28 w=4.6924	(9)
New page
Mx 	20.6362
My 	32.1451
T	63 w=4.6924	(\)
Mx 	25.3286
T	72 77 72 70 87 w=19.6387	(eject)
Mx 	47.6109
T	87 75 72 81 w=19.0967	(then)
Mx 	69.3512
T	86 87 88 747 w=18.8232	(stuff)
Mx 	90.8180
T	77 88 86 87 w=15.1904	(just)
Mx 	108.6520
T	83 68 86 87 w=17.1924	(past)
Mx 	104.3179
My 	176.1451
T	87 75 72 w=13.5791	(the)
Mx 	119.8055
T	74 76 79 79 86 w=16.8701	(gills)
Mx 	73.2094
My 	187.1052
T	20 19 w=9.3848	(10)
New page
Mx 	12.2343
My 	24.3326
My 	36.3326
New page
Mx 	20.6362
My 	32.1451
T	63 w=4.6924	(\)
Mx 	25.3286
T	82 83 72 81 w=20.4639	(open)
Mx 	45.7925
T	16 w=3.3691	(-)
Mx 	49.1616
T	86 83 85 72 68 71 w=27.5293	(spread)
Mx 	76.6909
T	62 w=3.0762	([)
Mx 	79.7671
T	82 71 71 w=15.4297	(odd)
Mx 	95.1968
T	32 w=4.3896	(=)
Mx 	99.5864
T	73 68 79 86 72 w=18.8916	(false)
Mx 	118.4780
T	64 w=3.0762	(])
Mx 	20.6362
My 	44.1451
Mx 	73.2094
My 	187.1052
T	20 21 w=9.3848	(12)
New page
Mx 	12.2343
My 	32.1451
T	63 w=4.6924	(\)
Mx 	16.9267
T	82 83 72 81 w=20.4639	(open)
Mx 	37.3906
T	16 w=3.3691	(-)
Mx 	40.7597
T	86 83 85 72 68 71 w=27.5293	(spread)
Mx 	68.2890
T	62 w=3.0762	([)
Mx 	71.3652
T	71 82 88 69 79 72 w=27.9346	(double)
Mx 	99.2998
T	32 w=4.3896	(=)
Mx 	103.6894
T	73 68 79 86 72 w=18.8916	(false)
Mx 	122.5810
T	64 w=3.0762	(])
Mx 	12.2343
My 	44.1451
Mx 	64.8075
My 	187.1052
T	20 22 w=9.3848	(13)
New page
Mx 	20.6362
My 	24.3326
New page
Mx 	12.2343
My 	32.1451
T	63 w=4.6924	(\)
Mx 	16.9267
T	82 83 72 81 w=20.4639	(open)
Mx 	37.3906
T	16 w=3.3691	(-)
Mx 	40.7597
T	86 83 85 72 68 71 w=27.5293	(spread)
Mx 	64.8075
My 	187.1052
T	20 24 w=9.3848	(15)
End page
Finish
